its/projects/UcfgNoProtobuf/Class1.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+// Test project used by the test "ucfgs_not_created_when_rules_not_enabled"
+
+namespace UcfgNoProtobuf
+{
+    public class Class1
```

Comment:
I think there should be at least one method here to be sure that nothing is generated